openpgp/packet/signature.go

```diff
@@ -518,11 +518,10 @@ func parseSignatureSubpacket(sig *Signature, subpacket []byte, isHashed bool) (r
 		}
 	case keyFlagsSubpacket:
 		// Key flags, section 5.2.3.21
```

Comment:
Not for this MR but rather for a future refactor. I noticed that RFC4880 has different sections than the [crypto-refresh](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-13.html#section-5.2.3.29). Thus, it can be a bit confusing to reference sections in the doc comments.

Reply:
Yeah, I agree. In general we should update all the references once RFC9580 comes out.